scripts/ups_as_envs.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+
+import os
+import json
+import sys
+
+for ups in json.loads(os.environ.get("VCAP_SERVICES", "{}")).get("user-provided", []):
+    for k, v in ups.get("credentials", {}).items():
+        sys.stdout.write("export %s='%s'\n" % (k, v.replace("'", "'\\''")))
```

Comment:
why `eval` these `stdout.write` instead of `os.environ[] = ""`

Reply:
Pretty sure that modifying `os.environ[]` will only modify the env variables for this process, not it's parent, which is the one that launches the actual server.

So for example that might work if we put this script inside of the launched app, but I kind of like it external - as it will work across non-Python apps that way.

(and the man page for modifying `os.environ[]` had enough caveats and scary warnings that it scared me off: <https://docs.python.org/2/library/os.html#process-parameters>)

Reply:
Makes sense